endpoint/http/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/http/request/mapper/AbstractRequestMapper.java

```diff
@@ -156,6 +159,59 @@ protected <T> T parseBody(HttpRequest httpRequest, Class<T> type) throws Invalid
     }
 
 
+    /**
+     * Deserializes HTTP body multipart form data.
+     *
+     * @param httpRequest HTTP request
+     * @param contentType the multipart contentType containing the boundary
+     * @return deserialized payload
+     * @throws InvalidRequestException if deserialization fails
+     * @throws IllegalArgumentException if httpRequest is null
+     */
+    protected Map<String, BlobValue> parseMultiPartBody(HttpRequest httpRequest, ContentType contentType) throws InvalidRequestException {
+        Ensure.requireNonNull(httpRequest, "httpRequest must be non-null");
+        Map<String, BlobValue> map = new HashMap<String, BlobValue>();
+        try {
+            MultipartStream multipartStream = new MultipartStream(
+                    new ByteArrayInputStream(httpRequest.getBody()),
+                    contentType.getParameter("boundary").getBytes());
```

Comment:
Where does "boundary" come from? Is this part of an RFC? If so, there probably should be a constant in some HTTP library to re-use.